yw_website/website/templates/registration/register.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+{% load staticfiles %}
+{% include 'includeable/navbar.html' %}
```

Comment:
same as other navbar. we probably want this to extend `__base_template.html` since that includes the navbar anyways